src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModelFactory.cs

```diff
@@ -118,11 +92,32 @@ public PackageModel Create(PackageSearchMetadataContextInfo metadata, ContractIt
             }
             else
             {
-                PackageMetadataRetrievalAdapter packageMetadataRetrievalAdapter = new PackageMetadataRetrievalAdapter(_searchService, metadata.Identity!, _packageSources, _includePrerelease);
-                IDeprecationCapable deprecationCapable = new DeprecationPackageMetadataCapability(packageMetadataRetrievalAdapter);
-                VulnerablePackageMetadataCapability vulnerableCapability = new VulnerablePackageMetadataCapability(packageMetadataRetrievalAdapter);
+                // Transitive dependencies are only available in the Installed tab
+                if (metadata.TransitiveOrigins != null && itemFilter.Equals(ContractItemFilter.Installed))
```

Comment:
We shouldn't need to specify the tab since a package that is transitive is defined by it having transitive origins.

```suggestion
                if (metadata.TransitiveOrigins != null)
```